portfolio/src/main/webapp/script.js

```diff
@@ -14,7 +14,8 @@
 
 /**
  * Adds a random greeting to the page.
- */
+*/
+
```

Comment:
I'd suggest reverting this change; the asterisk alignment in the original was fine, and it's pretty standard to have function comments right up against the function they're documenting.

Reply:
Got it, I'm not sure why I made that change!